### PR TITLE
Add appcast to grandperspective

### PR DESCRIPTION
**Pull request by tsparber**
Tick #16685

### CODE REVIEW

File: Casks/grandperspective.rb [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
+  url "http://downloads.sourceforge.net/project/grandperspectiv/grandperspective/#{version}/GrandPerspective-#{version.dots_to_underscores}.dmg"
+  appcast 'http://sourceforge.net/projects/grandperspectiv/rss?path=/grandperspective',
+          checkpoint: 'aa3a708f08a2b7f02fd269062bdda983d17c36007bb5fe46b1e11ab3dea07769'
   name 'GrandPerspective'
   homepage 'http://grandperspectiv.sourceforge.net/'
   license :gpl
 
   app 'GrandPerspective.app'
 
-  zap delete: '~/Library/Preferences/net.sourceforge.grandperspectiv.plist'
```
Comment (jawshooah) [inline Casks/grandperspective.rb]: Why the change here? `:trash` and `:delete` are synonyms. We don't yet support moving things to the trash bin.

Comment (tsparber) [inline Casks/grandperspective.rb]: I thought there were some discussions about using `trash:` for files which are maybe still important to the user (even when zapping) and `delete:` for files which are automatically created and "not so important". (I'm aware that its an alias). (like `trash` for preferences, delete for `caches...`). I wouldn't mind reverting this change.

**Comment by adidalal** [comment]
`:trash` could be as simple as implementing this bash function `trash () { command mv "$@" ~/.Trash ; }`

File: Casks/grandperspective.rb (hunk 1 above)
Comment (adidalal) [inline Casks/grandperspective.rb]: `:trash` could be as simple as implementing this bash function `trash () { command mv "$@" ~/.Trash ; }`

Comment (tsparber) [inline Casks/grandperspective.rb]: This could be as simple as that, but upon reading the interwebs, I found that such a simple approach would have some drawbacks. Misses handling of duplicate files, Finders "put back" option is also not available. I think the "put back" option would be one of the most important features (The user most likely didn't know where the file was before).
Using [NSFileManager trashItemAtURL](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSFileManager_Class/#//apple_ref/occ/instm/NSFileManager/trashItemAtURL:resultingItemURL:error:) seems to be a better approach. I don't know how hard it is to call Objective-C functions from Ruby, but this (or some wrapper) seems to be the cleanest approach.

Comment (jawshooah) [inline Casks/grandperspective.rb]: @adidalal [It's not quite that simple.](https://github.com/morgant/tools-osx/blob/master/src/trash)

**Comment by vitorgalvao** [comment]
Why the close?

**Comment by tsparber** [comment]
My bad, was cleaning up my branches from command line. I will double check next time.

File: Casks/grandperspective.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/grandperspective.rb]: Should we merge this, or change it back to `delete`? I’m fine either way.

**Comment by jawshooah** [comment]
Doesn't matter I suppose. Not worth holding up the merge.

**Comment by bazooka241** [comment]
Iｍ sorry . I can’t understand English.
So Im calling trancerater.
Thank you very much.

> H28/01/24 1:48、Josh Hagins <notifications@github.com> のメール：
> 
> oesn't matter I suppose. Not worth holding up the merge
